hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieCompactor.java

```diff
@@ -172,18 +172,12 @@ public static void main(String[] args) {
       throw new HoodieException("Fail to run compaction for " + cfg.tableName + ", return code: " + 1);
     }
     final JavaSparkContext jsc = UtilHelpers.buildSparkContext("compactor-" + cfg.tableName, cfg.sparkMaster, cfg.sparkMemory);
-    int ret = 0;
-    try {
-      ret = new HoodieCompactor(jsc, cfg).compact(cfg.retry);
-    } catch (Throwable throwable) {
-      throw new HoodieException("Fail to run compaction for " + cfg.tableName + ", return code: " + ret, throwable);
-    } finally {
-      jsc.stop();
-    }
-
+    int ret = new HoodieCompactor(jsc, cfg).compact(cfg.retry);
     if (ret != 0) {
       throw new HoodieException("Fail to run compaction for " + cfg.tableName + ", return code: " + ret);
     }
+    LOG.info("Success to run compaction for " + cfg.tableName);
+    jsc.stop();
```

Comment:
Could you help me understand why remove try-catch block here? 
If L175 failed and threw an exception then L180 would not be executed

Reply:
L175 will calls `UtilHelpers.retry` internal, which has try-catch block. This try-catch block is unreachable.